samples/sdl/main.c

```diff
@@ -18,10 +18,10 @@
 #include "string.h"
 #include <SDL.h>
 
-int _exit(int x)
+int Reboot(int x)
```

Comment:
You can also improve the function signature now: `static void Reboot()` or possibly `static void Reboot(unsigned int step)` (where `step` should be printed / have a unique number).